src/components/routes/Trips/Trips.tsx

```diff
@@ -76,60 +76,97 @@ class Trips extends React.Component<Props, State> {
               }
               return (
                 <>
-                  {isSubmitting && <CryptoPortal />}
-                  {!!upcoming.length && (
-                    <section>
-                      <AppConsumer>
-                        {({ screenType }: AppConsumerProps) => {
-                          const mobile = screenType < ScreenType.TABLET;
-                          return (
+                  <AccountNav config={[
+                    {
+                      title: 'Current',
+                      to: '/trips/current',
+                    },
+                    {
+                      title: 'Upcoming',
+                      to: '/trips/upcoming',
+                    },
+                    {
+                      title: 'Past',
+                      to: '/trips/past',
+                    },
+                    {
+                      title: 'Cancelled/\nRejected',
+                      to: '/trips/cancelled',
+                    }
+                  ]} />
+                  <div className="trip-cards-container">
+                    <Switch>
+                      <Route exact path="/trips/current" component={() =>
```

Comment:
when visiting /trips, if there are current trips, show current trip, else show upcoming.

The upcoming tab should have an empty state like "Book your next travel now"

Reply:
updated to use the empty default: 

![image](https://user-images.githubusercontent.com/18321302/51640301-f1073100-1f17-11e9-9f6b-2469c1f5ecce.png)

Reply:
👍